common/presenters/date-table/population-helpers.test.js

```diff
@@ -49,7 +53,7 @@ describe('population helpers', function () {
     i18n.t.restore()
   })
 
-  describe('freeSpaceCellContent', function () {
+  describe.skip('freeSpaceCellContent', function () {
```

Comment:
This test should be removed, as the approach has been replaced with the cell renderer concept.